app/helpers/email_helper.rb

```diff
@@ -90,4 +90,13 @@ def percentage_for(poll, index)
       (100 * poll.stance_counts[index].to_f / poll.stance_counts.max).to_i
     end
   end
+
+  def stance_choice_percentage_for(stance, stance_choice)
+    scores = stance.stance_choices.pluck(:score)
```

Comment:
Just that I'm pretty sure doing `MAX(column)` in postgres is way faster than doing `Array.max` in ruby